tests/test_simpleButler.py

```diff
@@ -204,11 +204,8 @@ def testDatasetImportTwice(self):
             butler2 = self.makeButler(writeable=True)
             # Import it once.
             butler2.import_(filename=file.name)
-            # Import it again, but ignore all dimensions
-            dimensions = set(
-                dimension.name for dimension in butler2.registry.dimensions.getStaticDimensions()
-            )
-            butler2.import_(filename=file.name, skip_dimensions=dimensions)
+            # Import it again
+            butler2.import_(filename=file.name)
```

Comment:
I wonder why the code coverage tool does not think this line ran...

Reply:
Good catch!  It's right; I tried sabotaging it and it seems to always be skipped.

At the top of the base test case in this file, we have:
```
datasetsIdType = int
```
and then this test method has:
```
if self.datasetsIdType is not uuid.UUID:
    self.skipTest("This test can only work for UUIDs")
```
but at the bottom we have two derived test cases that both set
```
datasetsIdType = uuid.UUID
```
so it's not clear to me what's going on.  I'll see if I can figure it out, but ideas are most welcome.

Reply:
Ha, nope, nothing so complicated as that.  I had accidentally renamed the test method to `tesImportTwice`.

Reply:
😄 On the plus side code coverage caught it.